cmd/cluster_phase_etcd.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/xetys/hetzner-kube/cmd/phases"
+	"github.com/xetys/hetzner-kube/pkg"
+)
+
+func init() {
+	command := declarePhaseCommand("etcd", "setups a etcd cluster", func(cmd *cobra.Command, args []string) {
+		provider, clusterManager, coordinator := getCommonPhaseDependencies(6, cmd, args)
+		keepData, _ := cmd.Flags().GetBool("keep-data")
+
+		phase := phases.NewEtcdSetupPhase(clusterManager, provider, phases.EtcdSetupPhaseOptions{KeepData: keepData})
+
+		if phase.ShouldRun() {
+			err := phase.Run()
+			FatalOnError(err)
```

Comment:
Maybe I'm a bit ripetitive but using RunE and return error when we should stop command is a better approach than use custom function like `FatalOnError` ;)